src/lib/authentication/authenticationSlice.ts

```diff
@@ -159,3 +160,4 @@ export const authSlice = createSlice({
 export const { tokenReceived, loggedOut } = authSlice.actions;
 
 export const tokenSelector = (state: RootState) => state.auth.data?.token;
+export const userSelector = (state: RootState) => state.auth.data?.userInfo;
```

Comment:
define selectors in Slice.

https://redux-toolkit.js.org/api/createSlice#selectors